recipe/meta.yaml

```
@@ -31,6 +31,10 @@ requirements:
     - pyyaml
     - setuptools >=40.6.0
     - wheel
+  run_constrained:
+    # [pandas] requires modin==0.28.1 and pyarrow (no pinning)
+    - pandas <3.0.0,>=1.0.0
+    - modin ==0.28.1
```

```suggestion
    - modin ==0.28.1
    - opentelemetry-api >=1.0.0,<2.0.0
    - opentelemetry-sdk >=1.0.0,<2.0.0
```
https://github.com/snowflakedb/snowpark-python/blob/v1.17.0/setup.py#L130-L133